perllib/FixMyStreet/Integrations/ExorRDI.pm

```diff
@@ -198,6 +208,14 @@ sub construct {
         0, 0, 0 # error counts, always zero
     );
 
+    # Mark all these problems are having been included in an RDI
+    my $now = DateTime->now->strftime( '%Y-%m-%d %H:%M' );
+    $problems->reset;
+    while ( my $report = $problems->next ) {
+        $report->set_extra_metadata('rdi_processed' => $now);
```

Comment:
Do we want to do this if the RDI file has been generated via the admin page, rather than by email?